### PR TITLE
Blog on RedHat Virtualization event

### PR DESCRIPTION
**Pull request by SachinRammoorthy**
Wrote a blog post on the _[Virtualization as a path to an optimized, modern datacenter](https://www.redhat.com/en/events/virtualization-path-optimized-modern-datacenter)_ webinar.

Part of [this](https://codein.withgoogle.com/dashboard/task-instances/5446356657242112/) task.

Tweet: https://twitter.com/sachin27_02/status/951134718090559488

### CODE REVIEW

File: _posts/2018-01-10-sachin-virtualization.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "JBoss Developer Event on Virtualization"
+description: "An enlightening experience at the RedHat webinar today about Virtualization and its benefits"
```
Comment (aashutoshrathi) [inline _posts/2018-01-10-sachin-virtualization.md]: remove todau from this line

File: _posts/2018-01-10-sachin-virtualization.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+headline: 
+modified: 2018-01-10 22:20:23 +0000
+category: personal
+tags: []
+feature-img:
```
Comment (aashutoshrathi) [inline _posts/2018-01-10-sachin-virtualization.md]: If possible add banner here

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi updated blog.

File: _posts/2018-01-10-sachin-virtualization.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+I attended the webinar with very little knowledge of what Virtualization actually meant. I learnt that it optimized digital transmission, was at the peak of innovation in this modern era, and increased the competitive capability of any company. It essentially modernizes infrastructure and applications, and is a key enabler in helping us on our digital transmission journey. As the presenter explained to me in relatively simple words, virtualization means to create a virtual version of a device or resource, like a server or a network where the framework divides the resource into one or more execution environments.
+
+Most people actually think virtualization has occurred everywhere and this period has now rolled over, but in reality, this is not the case at all. RedHat conducted surveys where they found out that virtualization is actually on the rise! In fact, it is expected to rise by over 20% in deployment over the next 2 years.
```
Comment (aashutoshrathi) [inline _posts/2018-01-10-sachin-virtualization.md]: `Most people actually think virtualization has occurred everywhere` is not making much sense, make it 
`Most people actually think virtualization is being adapted everywhere`

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi have updated it.

**Pull request merged**